dictionaries/en_shared/src/acronyms.txt

```diff
@@ -1,10 +1,12 @@
 # Acronyms, and Initialisms
-ASAP
-FOMO
-IMAX
-lidar
-lidars
-TLDR
-usd
-USD
-YOLO
+ASAP # as soon as possible
+BDSM # Sexual practice
+FOMO # fear of missing Out
+IMAX #  Image MAXimum
+LIDAR # Light Detection And Ranging
```

Comment:
`lidar` is allowed in many spelling contexts. So, it is better to keep it lowercase. See: https://en.wikipedia.org/wiki/Lidar

Reply:
I hesitated here, I made back and forth. I shouldn't have committed it

Reply:
no worries.